.mk/development.mk

```diff
@@ -22,6 +22,21 @@ undeploy-loki: ## Undeploy loki.
 	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml | kubectl --ignore-not-found=true  delete -f - || true
 	-pkill --oldest --full "3100:3100"
 
+.PHONY: deploy-kafka
+deploy-kafka:
+	@echo -e "\n==> Deploy kafka"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
+	kubectl create -f "https://strimzi.io/install/latest?namespace="$(NAMESPACE) -n $(NAMESPACE)
+	kubectl create -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka-cluster.yaml" -n $(NAMESPACE)
+	kubectl create -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka-topic.yaml" -n $(NAMESPACE)
```

Comment:
Strimzi only provide this endpoint for its latest version.

The other option is to track this file in one of our repository, but this file has to be changed depending of the targeted namespace.

Since we only use this for now as quick setup for development and since this file is quite big (13k lines) would you be fine to keep using latest for know? If at one point we experience issues it will still be possible to freeze the version there.

Reply:
Well if there is no easy solution let's keep this as is. Thanks